hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/ClientStatisticsCodec.java

```diff
@@ -67,7 +65,7 @@
  * lastStatisticsCollectionTime: The time stamp (milliseconds since epoch) when the latest update for the statistics is
  * collected.
  * 
- * Near cache statistics (see {@link com.hazelcast.monitor.NearCacheStats}):
+ * Near cache statistics (see {@link NearCacheStats}):
```

Comment:
I believe this needs a change in the client protocol definition too, otherwise, the next generation will override it.

Reply:
The protocol definition is this: https://github.com/hazelcast/hazelcast-client-protocol/blob/master/protocol-definitions/Client.yaml#L647
No need to do it now, I need to touch this codec anyways with the metric work. A note for myself.